pom.xml

```diff
@@ -222,7 +222,8 @@ under the License.
     <dependency>
       <groupId>org.apache.maven.shared</groupId>
       <artifactId>maven-dependency-analyzer</artifactId>
-      <version>1.11.1</version>
+      <!-- NEED TO WAIT FOR RELEASE -->
+      <version>1.11.2-SNAPSHOT</version>
```

Comment:
I've pushed this version to the snapshots repo. I'm checking on whether we can commit this based on that. I see at least one other plugin repo depending on a snapshot version.

Reply:
given that we can depend on snapshot versions, are you comfortable with this PR being merged now?

Reply:
Yes but I'm hoping to get @bimargulies  to review it first so he has more of my work to base his feedback off

Reply:
@elharo The PR can be merged now